pkg/assembler/backends/ent/dependency.go

```diff
@@ -42,13 +44,15 @@ type Dependency struct {
 type DependencyEdges struct {
 	// Package holds the value of the package edge.
 	Package *PackageVersion `json:"package,omitempty"`
-	// DependentPackage holds the value of the dependent_package edge.
-	DependentPackage *PackageName `json:"dependent_package,omitempty"`
+	// DependentPackageName holds the value of the dependent_package_name edge.
+	DependentPackageName *PackageName `json:"dependent_package_name,omitempty"`
+	// DependentPackageVersion holds the value of the dependent_package_version edge.
+	DependentPackageVersion *PackageVersion `json:"dependent_package_version,omitempty"`
 	// loadedTypes holds the information for reporting if a
 	// type was loaded (or requested) in eager-loading or not.
-	loadedTypes [2]bool
+	loadedTypes [3]bool
```

Comment:
Question for my understanding: What is the reason for the change from 2 to 3 here?

Reply:
I see line 87 below. Could we instead make these bools separate variables with clear names?

Reply:
This code is generated by Ent so I'm not so sure we have room for customization.
At the same time, the fact that they're generated from Ent could also mean we won't have to work on them directly so having unclear names isn't a big problem.

WDYT?

Reply:
Makes sense, I didn't realzie that. Resolving